### PR TITLE
Added helm chart for NATS

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Copied local NATS helm charts from [kyma](https://github.com/kyma-project/kyma/tree/main/resources/eventing/charts/nats). (First commit)
- There are no major changes in upstream NATS helm chart. Only `PodDisruptionPolicy` is introduced. It will be discussed later whether we want to include `PodDisruptionPolicy`.
- Added `accounts-secret.yaml` for accounts.

Test: `helm install --namespace=kyma-system eventing .` is working.

**Related issue(s)**

- https://github.com/kyma-project/nats-manager/issues/11

### CODE REVIEW

File: resources/nats/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+To install this chart, follow these steps:
+
+1. Install NATS into the `nats` namespace using Helm 3 :
+```bash
+kubectl create namespace nats
+helm template nats nats -n nats | kubectl apply -f -
+```
+2. Test the installation:
+```bash
+$ kubectl -n nats port-forward nats-1 4222
```
Comment (marcobebway) [inline resources/nats/README.md:21]: (nit) How about `nats-0` instead?

File: resources/nats/values.yaml [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+###############################
+istio:
+  enabled: false
+
+###############################
+#                             #
+#  NATS Server Configuration  #
+#                             #
+###############################
+nats:
```
Comment (marcobebway) [inline resources/nats/values.yaml:63]: Does NATS upstream use this in their charts too?

Comment (mfaizanse) [reply resources/nats/values.yaml:63]: yes, its mentioned [here](https://github.com/nats-io/k8s/blob/fc718ec58e8dcf2334effa518388dd8ba0b6a3ce/helm/charts/nats/values.yaml#L6).

File: resources/nats/README.md (hunk 1 above)
Comment (mfaizanse) [reply resources/nats/README.md]: updated

**Review by marcobebway — APPROVED** [review]

**Pull request merged**